packages/workflow-core/src/lib/workflow-runner.ts

```diff
@@ -240,7 +240,12 @@ export class WorkflowRunner {
       stateNames: iterarivePluginParams.stateNames,
       //@ts-ignore
       iterateOn: this.fetchTransformers(iterarivePluginParams.iterateOn),
-      action: (context: TContext) => actionPlugin!.invoke(context, this.#__config),
+      action: (context: TContext) =>
+        actionPlugin!.invoke({
+          ...context,
+          workflowRuntimeConfig: this.#__config,
+          workflowRuntimeId: this.#__runtimeId,
+        }),
```

Comment:
Here I add `workflowRuntimeConfig` and `workflowRuntimeId` into the context of the plugin transformer, alongside the workflow's context.
This change helps removing the `workflowRuntimeId` we used to have in the machine context (line 128).